client/modules/IDE/components/Preferences.jsx

```diff
@@ -151,10 +154,9 @@ class Preferences extends React.Component {
                   aria-atomic="true"
                   value={this.state.fontSize}
                   onChange={this.onFontInputChange}
+                  type="text"
                   ref={(element) => { this.fontSizeInput = element; }}
-                  onClick={() => {
-                    this.fontSizeInput.select();
-                  }}
+                  onClick={() => { this.fontSizeInput.select(); }}
```

Comment:
could this this line just be
```jsx
onClick={this.fontSizeInput.select}
```

Reply:
@catarak It shows an error if you do so!

Reply:
oh oh oh it's because it's a `ref`, got it 😸

Reply:
@catarak Yeah exactly!
Can this be merged now? :)